src/panels/lovelace/cards/hui-stack-card.ts

```diff
@@ -92,6 +92,7 @@ export abstract class HuiStackCard<T extends StackCardConfig = StackCardConfig>
     return css`
       .card-header {
         color: var(--ha-card-header-color, var(--primary-text-color));
+        text-align: var(--ha-card-header-text-align, start);
```

Comment:
The variable sounds like it is for `ha-card` but you implemented it just for stack cards. I would expect this to be at least also implemented in `ha-card`.

Reply:
Better like this?